WORKSPACE

```diff
@@ -35,10 +35,10 @@ rules_foreign_cc_dependencies()
 
 http_archive(
     name = "capnp-cpp",
-    sha256 = "7ccdf8eeed127d439bc8a94d5742f87d3e5e305f13c3bbb2797005eb6705bf98",
-    strip_prefix = "capnproto-capnproto-d0e3946/c++",
+    sha256 = "f8487168babfae11132c065c7e78d476385de0cbcb2c4024427cbcb839bcdd57",
```

Comment:
The new capnp commit is 255da5372946dbe9fc812347d3b38fc613eb9714

Reply:
We'll probably need to change this again anyway (I needed to rebase your commit as I was getting issues with `allowCancellation` otherwise).